### DISCUSSION TITLE
compile error in idea

### DISCUSSION THREAD

**Discussion by robinhe87** [dbeaver/dbeaver#13058] [DBeaver Development]
hi,I clone the project, and then copy *.iml from dbeaver-idea-project to dbeaver，but it dosn't work.the erros follow image:
![image](https://user-images.githubusercontent.com/37928952/113258052-04836300-92fe-11eb-97b0-f6a844176b43.png)

can you help me,tks.

**Comment by uslss** [comment]
Hi @robinhe87 ,
could you try to do it accordingly to [guide](https://github.com/dbeaver/dbeaver/wiki/Develop-in-IDEA)?
